orttraining/orttraining/core/framework/gradient_graph_builder.h

```diff
@@ -77,6 +77,13 @@ static std::unordered_map<std::string, std::unordered_set<size_t>>
 
 static std::unordered_set<std::string> INVERTIBLE_OPS{"LayerNormalization",
                                                       "Relu"};
+static std::unordered_set<int64_t> CAST_GRAD_ALLOWED_TYPES{
```

Comment:
Since this variable is not used anywhere other than `gradient_graph_builder.cc`, shall we only define it there to avoid making unnecessary static copies of the `unordered_map` in different translation units? We can also drop the `static`, since it is only ever used once or twice per session (it can be a const initialized during `GradientGraphBuilder` construction)?

Reply:
moved under the class definition